ApplyToBecomeInternal/ApplyToBecomeInternal.Tests/Pages/TaskList/Decision/DecisionDateIntegrationTests.cs

```diff
@@ -91,30 +103,30 @@ public async Task Should_redirect_onSubmit()
 			var request = new AdvisoryBoardDecision
 			{
 				Decision = AdvisoryBoardDecisions.Approved,
-				AdvisoryBoardDecisionDate = new DateTime(2021, 01, 01),
+				AdvisoryBoardDecisionDate = new DateTime(DateTime.Today.Year, 01, 01),
```

Comment:
Do you need the Month & Day if using DateTime.Today.Year?

Reply:
Yes; all three components are required when initialising a DateTime object. The `DateTime.Today.Year` just makes the year part dynamic so that this doesn't break due to the new "Year must be this year" requirement.